pypeit/spectrographs/gemini_gmos.py

```diff
@@ -565,13 +565,13 @@ def list_detectors(self, mosaic=False):
             return np.array([self.get_det_name(_det) for _det in self.allowed_mosaics])
 
         return np.array([detector_container.DetectorContainer.get_name(i+1)
-                             for i in range(self.ndet)]).reshape(2,-1)
+                             for i in range(self.ndet)]).reshape(3,1)
```

Comment:
This is the actual change. Instead of 2 rows of detectors in the spectral direction, like DEIMOS, GMOS has 3 detectors covering the whole dispersed spectrum. This should return

```
array([['DET01'],
       ['DET02'],
       ['DET03']], dtype='<U5')
```